livekit_analytics.proto

```diff
@@ -19,6 +19,19 @@ enum StreamType {
   DOWNSTREAM = 1;
 }
 
+message AnalyticsStream {
```

Comment:
@shishirng @davidzhao Thinking of this model for per stream stats. If simulcast, will be sending one of these per available layer. That means, this may be have missing samples as layers could be turned on/off.

The track level stuff will be an aggregate of these (except for jitter and delay which will be the max of these).

Just wanted to have granular analytics which we can use however we see fit while processing them. It gives the full picture.

For SVC codecs (VP9, etc.), there is only one stream with layers interleaved in them. So, those kind of codecs will not have all of these filled out at layer level. The things that make sense at layer level for those codecs are
```
total_packets
total_bytes
total_frames
```
Rest are not applicable at layer level and they will be in the track level stats.

What do you think?

Reply:
Actually, `delay` does not make sense at all. `rtt` makes more sense. But, `rtt` will be available for down tracks only (i. e. it can be calculated on media sender side using RTCP receiver report received from the other side). For up tracks, we won't have RTT. We can probably use some other mechanism (like using STUN ping time or just use down track RTT for up track also).

I will drop populating `delay` in the analytics message.

Reply:
@boks1971 this looks good